.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,50 @@
+name: Performance Test
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'K6 Load Test'  
+
+jobs:
+  k6_test_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+ 
+      - name: Install K6
+        run: |
+          wget https://github.com/grafana/k6/releases/download/v0.38.2/k6-v0.38.2-linux-amd64.tar.gz
+          tar -xvf k6-v0.38.2-linux-amd64.tar.gz k6-v0.38.2-linux-amd64/k6
+          mv k6-v0.38.2-linux-amd64/k6 .
+          ./k6
+
+      - name: Make dev-env
+        run: |
+          mkdir $HOME/.kube
+          make dev-env
+          echo "sleeping 5secs to get controller podname .." && sleep 5
+          echo "now waiting for controller pod to be Running.."
+          kubectl wait pod -n ingress-nginx --for condition=Ready `kubectl get po -n ingress-nginx | grep controller | awk '{print $1}'`
+          kubectl get all -A
+
+      - name: Deploy workload
+        run: |
+          kubectl create deploy k6 --image nginx:alpine --port 80 && kubectl expose deploy k6 --port 80 && kubectl create ing k6 --class nginx --rule k6.ingress-nginx.local/=k6:80
+          echo "sleeping 5secs to get workload podname.." && sleep 5
+          echo "now waiting for workload pod to be Running.."
+          kubectl wait pod --for condition=Ready  `kubectl get po | grep k6 | awk '{print $1}'`
+          kubectl get all,ing
+
+      - name: Test with k6
+        run: |
+          set -x
```

Comment:
Do this prints some token it shouldn't in logs?

Reply:
https://github.com/longwuyuan/k6-loadtest-example/runs/6445012501?check_suite_focus=true  I didn't see any tokens being printed. The idea was to show the command executed just as a separator. But I can remove the `set -x` it if you advise.

Reply:
I removed all the echo statements. Please check